### PR TITLE
fix: problem with catalyst helper function

### PR DESCRIPTION
**Pull request by menduz**
# What? 
Fixes a problem by deleting it

# Why? 
...

### CODE REVIEW

**Review by pbosio — APPROVED** [review]

File: packages/decentraland-loader/lifecycle/controllers/parcel.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export class ParcelLifeCycleController extends EventEmitter {
   public resetScenesOnSight() {
     this.currentlySightedParcels.clear()
 
-    this.parcelStatus.forEach((value: ParcelLifeCycleStatus, key: string) => {
+    this.parcelStatus.forEach((value: ParcelLifeCycleStatus, _key: string) => {
```
Comment (gonpombo8) [inline packages/decentraland-loader/lifecycle/controllers/parcel.ts:35]: thanku

**Review by gonpombo8 — APPROVED** [review]

**Comment by menduz** [comment]
I pushed a change to reflect ADR53

Also, gon. In regards to catalyst client: I believe we are currently using only two functions that can be replaced with a plaing `fetch`. You may find interesting that we can delete the catalyst-client usage from this repo @gonpombo8

**Comment by gonpombo8** [comment]
@menduz nice. Issue created https://github.com/decentraland/sdk/issues/169 🎉

**Review by gonpombo8 — APPROVED** [review]

**Comment by menduz** [comment]
@nachomazzara @pbosio we can use a second validation before merging please?

**Comment by pbosio** [comment]
> @nachomazzara @pbosio we can use a second validation before merging please?

It's working for me. The only thing worth mention is that with this changes, if your are in a test catalyst and you want to test wearables deployed in org alongside with wearables in testing catalyst you can't.
But I think that's not a common scenario at all... :)

**Pull request merged**